.env

```diff
@@ -0,0 +1,22 @@
+if [[ -d .venv ]]
+then
+    source ./.venv/bin/activate
```

Comment:
your upper suggestion , also meant ? 
```suggestion
    source "$VIRTUAL_ENV"
```

Reply:
I thought about that , but beauty of this thing exactly in a fact that you **don't need to do anything beside cd'ing to dir** , with variable approach I would need to type "VIRTUAL_ENV=./.venv/bin/activate" and than there is no big difference to type "source ./venv/bin/activate" :)

Reply:
hm, don't get it.
I just meant, you should not `source ./.venv/bin/activate` twice.